plugin/draft_editor/draft_editor.py

```diff
@@ -104,15 +104,26 @@ def get_editor_app() -> ui.App:
     raise RuntimeError("Draft editor is not running")
 
 
-def close_editor(submit_draft: bool):
-    global last_draft
+def close_editor(submit_draft: bool) -> None:
     remove_tag("user.draft_editor_active")
     actions.edit.select_all()
     selected_text = actions.edit.selected_text()
     actions.edit.delete()
     actions.app.tab_close()
-    actions.user.switcher_focus_window(original_window)
-    actions.sleep("300ms")
     if submit_draft:
+        global last_draft
```

Comment:
The community standard is to always have globals as the first line in the function